components/measurement/nettests/WebConnectivity.js

```diff
@@ -167,32 +167,48 @@ FailureString.propTypes = {
   failure: PropTypes.string
 }
 
+const DnsNarrowAnswerCell = (props) => (
+  <Box width={1/12}>{props.children}</Box>
+)
+
 const DnsAnswerCell = (props) => (
-  <Box width={1/8}>{props.children}</Box>
+  <Box width={1/4}>{props.children}</Box>
 )
 
 DnsAnswerCell.propTypes = {
   children: PropTypes.any
 }
 
-const FiveColRow = ({ name = 'Name', netClass = 'Class', ttl = 'TTL', type = 'Type', data = 'DATA', header = false}) => (
+const dnsAnswerIpInfo = (dnsAnswer) => {
+    return dnsAnswer.asn
```

Comment:
Nested ternaries are a bit hard to read, what about composing the string this way?

```
const asn = dnsAnswer.asn ? `AS${dnsAnswer.asn}` : 'Unknown AS'
cons asOrgName = dnsAnswer.as_org_name ? `(${dnsAnswer.as_org_name})` : ''

return `${asn} ${asOrgName}`.trim()
```

Reply:
Thanks for the suggestion.  I changed the code accordingly.